src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -15,13 +15,13 @@ export default View.extend({
       {{/if}}
       {{#if email}}
         <ul class="email-info ov-info">
-          <li>{{i18n code="oie.enroll.okta_verify.email" bundle="login" arguments="email"}}</li>
+          <li>{{{i18n code="oie.enroll.okta_verify.email.info" bundle="login" arguments="email"}}}</li>
```

Comment:
Do we need to add an explain comment here?

Reply:
We do this in many places and also we are explicitly escaping the phone I dont see a need for comment.